Challo/Challo/Modules/Trails/TrailListing/Views/TrailListingPage.swift

```diff
@@ -13,28 +13,31 @@ struct TrailListingPage: View {
     
     var cardList: some View {
         Group {
-            if presenter.isLoading {
-                Loading(isAnimating: .constant(true), style: .large)
-            } else {
-                ScrollView {
-                    VStack(spacing: 30) {
-                        ForEach(presenter.trailListingCards.indices, id: \.self) { index in
-                            NavigationLink(destination: presenter.trailProfilePage) {
-                                presenter.trailListingCards[index]
-                            }.buttonStyle(PlainButtonStyle())
-                            .simultaneousGesture(TapGesture().onEnded {
-                                presenter.populateTrailProfilePage(trailIndex: index)
-                            })
-                        }
-                    }.padding(EdgeInsets(top: 50, leading: 60, bottom: 30, trailing: 60))
+            RefreshableScrollView(refreshing: $presenter.isRefreshing) {
+                if presenter.isLoading {
+                    VStack(alignment: .trailing) {
+                        Loading(isAnimating: .constant(true), style: .large)
+                    }
```

Comment:
Hmm if `Loading` is inside the scrollview then it would be possible to scroll when it is loading. Maybe can shift `Loading` out of the scrollview